packages/@okta/vuepress-site/guides/sign-users-out/sign-out-of-okta/index.md

```diff
@@ -1,6 +1,7 @@
 ---
 title: Sign Users Out of Okta
 ---
-If you want users to be prompted for credentials the next time they try to sign in, sign them out of Okta as well by ending their session on the Okta authorization server.
+
+Sign users out of Okta by ending their session on the Okta authorization server.
```

Comment:
Sign users out of Okta by ending their session on the Okta Authorization Server.